src/QmlControls/ParameterEditorDialog.qml

```diff
@@ -138,6 +149,26 @@ QGCViewDialog {
                 }
             }
 
+            Column {
+                Repeater {
+                    id: bitmaskEditor
+                    model: fact.bitmaskStrings
+                    delegate : QGCCheckBox {
+                        text : modelData
+                        checked : fact.value & fact.bitmaskValues[index]
+                        function check_uncheck_all() {
+                            if (text.toLowerCase() == "all") {
```

Comment:
What I asked is that when you select "all" it actually selects all the values. I suggested a strcmp, but @DonLakeFlyer is right that it's too generic. Maybe you can do the same but looking at the value instead?